static/app/views/monitors/components/checkInTimeline.tsx

```diff
@@ -18,15 +23,15 @@ interface Props {
   width?: number;
 }
 
-function getAggregateStatus(envData: MonitorBucketEnvMapping) {
-  // Orders the status in terms of precedence for showing to the user
-  const statusOrdering = [
-    CheckInStatus.OK,
-    CheckInStatus.MISSED,
-    CheckInStatus.TIMEOUT,
-    CheckInStatus.ERROR,
-  ];
+// Orders the status in terms of precedence for showing to the user
+const statusOrdering = [
```

Comment:
probably can be a in a `constants.tsx`

```suggestion
const CHECKIN_STATUS_PRECEDENT = [
```